content/linkedin-scraper.js

```diff
@@ -120,5 +129,7 @@ async function scrapeLinkedInJobs() {
 
       sendProgressPercentage(overallProgressPercentage);
     }
+
+    await changePage(pageIndex+2);
```

Comment:
Hey @BillyDotWS, thanks for the PR! Just a quick suggestion: let's add a condition to avoid navigating to the next page if we're already on the last one. Appreciate your contribution!

```suggestion

    if (pageIndex < pageCount - 1) {
      await changePage(pageIndex + 2);
    }
```

Reply:
Good point, didn't notice this possibility, committed suggestion 👍

Reply:
Thanks